exchanges/coinbene/coinbene_websocket.go

```diff
@@ -0,0 +1,328 @@
+					newOB.Bids = bids
+					newOB.AssetType = orderbook.Swap
+					newOB.Pair = currency.NewPairFromString(strings.Replace(orderBook.Topic, "tradeList.", "", 1))
+					newOB.ExchangeName = c.Name
+					err = c.Websocket.Orderbook.LoadSnapshot(&newOB, true)
+					if err != nil {
+						c.Websocket.DataHandler <- err
+						continue
+					}
+					c.Websocket.DataHandler <- wshandler.WebsocketOrderbookUpdate{Pair: newOB.Pair,
+						Asset:    orderbook.Swap,
+						Exchange: c.Name,
+					}
+				} else if orderBook.Action == "update" {
+					newOB := wsorderbook.WebsocketOrderbookUpdate{
+						Asks:         asks,
+						Bids:         bids,
+						AssetType:    orderbook.Swap,
+						CurrencyPair: currency.NewPairFromString(strings.Replace(orderBook.Topic, "tradeList.", "", 1)),
+						UpdateID:     orderBook.Version,
+					}
+					err = c.Websocket.Orderbook.Update(&newOB)
+					if err != nil {
+						c.Websocket.DataHandler <- err
+						continue
+					}
+					c.Websocket.DataHandler <- wshandler.WebsocketOrderbookUpdate{Pair: newOB.CurrencyPair,
+						Asset:    orderbook.Swap,
+						Exchange: c.Name,
+					}
+				}
+			case strings.Contains(result[topic].(string), "kline"):
+				var kline WsKline
+				var tempFloat float64
+				var tempKline []float64
+				err = common.JSONDecode(stream.Raw, &kline)
+				if err != nil {
+					c.Websocket.DataHandler <- err
+					continue
+				}
+				for x := 2; x < len(kline.Data[0]); x++ {
+					tempFloat, err = strconv.ParseFloat(kline.Data[0][x].(string), 64)
+					if err != nil {
+						c.Websocket.DataHandler <- err
+						continue
+					}
+					tempKline = append(tempKline, tempFloat)
+				}
+				c.Websocket.DataHandler <- wshandler.KlineData{
+					Timestamp:  time.Unix(int64(kline.Data[0][1].(float64)), 0),
+					Pair:       currency.NewPairFromString(kline.Data[0][0].(string)),
+					AssetType:  orderbook.Swap,
+					Exchange:   c.Name,
+					OpenPrice:  tempKline[0],
+					ClosePrice: tempKline[1],
+					HighPrice:  tempKline[2],
+					LowPrice:   tempKline[3],
+					Volume:     tempKline[4],
+				}
+			case strings.Contains(result[topic].(string), "user.account"):
+				var userinfo WsUserInfo
+				err = common.JSONDecode(stream.Raw, &userinfo)
+				if err != nil {
+					c.Websocket.DataHandler <- err
+					continue
+				}
+				c.Websocket.DataHandler <- userinfo
+			case strings.Contains(result[topic].(string), "user.position"):
+				var position WsPosition
+				err = common.JSONDecode(stream.Raw, &position)
+				if err != nil {
+					c.Websocket.DataHandler <- err
+					continue
+				}
+				c.Websocket.DataHandler <- position
+			case strings.Contains(result[topic].(string), "user.order"):
+				var orders WsUserOrders
+				err = common.JSONDecode(stream.Raw, &orders)
+				if err != nil {
+					c.Websocket.DataHandler <- err
+					continue
+				}
+				c.Websocket.DataHandler <- orders
+			default:
+				c.Websocket.DataHandler <- fmt.Errorf("unhandled response '%s'", stream.Raw)
```

Comment:
Can change this to `fmt.Errorf("%s - unhandled response '%s'", c.Name, stream.Raw)`